packages/account/src/Components/Routes/__tests__/route-with-sub-routes.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { RouteWithSubRoutesRender } from '../route-with-sub-routes';
+import { Redirect } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    Route: jest.fn(({ exact, path }) => (
+        <div>
+            <span>Route loaded</span>
+            <span>{String(exact)}</span>
+            <span> {path}</span>
```

Comment:
```suggestion
            <span>Route loaded {String(exact)} {path}</span>
```
@likhith-deriv will it work the same way?🙂

Reply:
Will get the following error - `This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.`

Reply:
@likhith-deriv yeah, now when it's one element, expects have to be changed too, e.g. expect(screen.getByText('true') has to be used with RegExp instead.
Please try to search for this whole element with RegExp, it should work:
 line 41: `expect(screen.getByText(/route loaded \/test true/i)).toBeInTheDocument();`